src/main/resources/fileTemplates/config.jelly.ft

```diff
@@ -0,0 +1,17 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+Copyright DATE, Issuer.....
+
+Licensed under ...
+
+@Author
+-->
+
+<?jelly escape-by-default='true'?>
+<j:jelly xmlns:j="jelly:core" xmlns:f="/lib/form" >
+    <f:section title="${%Put Section Name Here}">
+
+        <!-- see https://www.jenkins.io/doc/developer/forms/ for help -->
```

Comment:
This isn't that useful I'm not sure if it should be showing up in every created file, I don't think other templates do that

Reply:
Happy to remove it